tests/queries/0_stateless/02157_url_basic_auth.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"net/http"
+	"github.com/gin-gonic/gin"
+)
+
+func main() {
+	accounts := gin.Accounts{
+		"admin1": "password",
+		"admin2": "password/",
```

Comment:
Worth adding username with some symbols that should be encoded.

Reply:
Also added username with special symbols as you ask.